lms/envs/common.py

```diff
@@ -2274,6 +2274,10 @@
     'terms_of_service': 'hidden',
     'city': 'hidden',
     'country': 'hidden',
+    'first_name': 'optional',
+    'last_name': 'optional',
+    'company': 'optional',
+    'title': 'optional',
```

Comment:
We should not add these here. These fields will only be used on certain microsites.